modules/packages/TOML.chpl

```diff
@@ -843,7 +894,8 @@ used to recursively hold tables and respective values
         when fieldDate do return 'datetime';
         when fieldToml do return 'toml';
         otherwise {
-          halt("Unknown type: '", this.tag, "'");
+          throw new TomlError("Unknown type");
+          return "nil";
```

Comment:
I believe this is due to an issue you were opening. It might be worth adding that issue number to the comment here (if it's open already).

Reply:
See PR description